localization/english/religion/wc_religion_l_english.yml

```
@@ -5,4 +5,502 @@
+ 
+ air_group:0 "Aerism"
+ air_group_adj:0 "Aerial"
+ air_group_adherent:0 "Aerist"
+ air_group_adherent_plural:0 "Aerists"
+ air_group_desc:0 "Among all the elements, the Aerists regard air as the main force and worship air elementals as deities."
+ 
+ alakir_worship:0 "Al'Akirism"
+ alakir_worship_adj:0 "Al'Akiric"
+ alakir_worship_adherent:0 "Al'Akirist"
+ alakir_worship_adherent_plural:0 "Al'Akirists"
+ alakir_worship_desc:0 "$alakir_worship_adherent_plural$ worship one of the Elemental Lords, Al'Akir the Windlord, as their main god. Other air elementals are seen as minor deities."
+ 
+ earth_group:0 "Geoism"
+ earth_group_adj:0 "Geo"
+ earth_group_adherent:0 "Geoist"
+ earth_group_adherent_plural:0 "Geoists"
+ earth_group_desc:0 "Among all the elements, the Geoists regard earth as the main force and worship earth elementals as deities."
+ 
+ therazane_worship:0 "Therazanism"
+ therazane_worship_adj:0 "Therazanic"
+ therazane_worship_adherent:0 "Therazanist"
+ therazane_worship_adherent_plural:0 "Therazanists"
+ therazane_worship_desc:0 "$therazane_worship_adherent_plural$ worship one of the Elemental Lords, Therazane the Stonemother, as their main god. Other earth elementals are seen as minor deities."
+ 
+ theradras_worship:0 "Theradrasism"
+ theradras_worship_adj:0 "Theradrasic"
+ theradras_worship_adherent:0 "Theradrasist"
+ theradras_worship_adherent_plural:0 "Theradrasists"
+ theradras_worship_desc:0 "Princess Theradras was an enormous earth elemental and a daughter of Therazane, the Stonemother. After mating with Zaetar, son of Cenarius, Theradras became the mother of the first centaur. The centaurs, seeing the disgust in their father's eyes, brutally killed Zaetar. And to this day, Theradras is the patron of her progeny."
+ 
+ fel_group:0 "Fel"
+ fel_group_adj:0 "Fel"
+ fel_group_adherent:0 "Fel Follower"
+ fel_group_adherent_plural:0 "Fel Followers"
+ fel_group_desc:0 "$fel_group_adherent_plural$ belive Chaos is the natural order of things. They master Fel magic, created as a result of the mutual destruction of Light and Void."
+ 
+ burning_legion_religion:0 "Sargerism"
+ burning_legion_religion_adj:0 "Sargerite"
+ burning_legion_religion_adherent:0 "Sargerite"
+ burning_legion_religion_adherent_plural:0 "Sargerites"
+ burning_legion_religion_desc:0 "Bent on destroying all life in the universe, Sargeras is seen as one of the greatest threats to creation, and in a smaller sense, Azeroth. At his beck and call during his burning crusade across the Great Beyond is a loyal, fel-infused legion of demons, the Burning Legion, ready to carry out the destruction Sageras so very much desires. Followers of the Dark Titan are only capable of cruelty and deceit, as they chase any chance to bring about the extinguishment of life."
```

Changed army to legion, as it feels a little redundant when the Burning Legion is used just a few words later. removed so very much desires, as the phrasing is quite awkward, and desires on its own works just as well.
```suggestion
 burning_legion_religion_desc:0 "Bent on destroying all life in the universe, Sargeras is seen as one of the greatest threats to creation, and in a smaller sense, Azeroth. At his beck and call during his burning crusade across the Great Beyond is a loyal, fel-infused army of demons, the Burning Legion, ready to carry out the destruction Sageras desires. Followers of the Dark Titan are only capable of cruelty and deceit, as they chase any chance to bring about the extinguishment of life."
```